nodejs/basic-bot/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const bodyparser = require('body-parser')
+const app = express();
+
+const PORT = process.env.PORT || 9000;
+
+app.use(bodyparser.urlencoded({ extended: false }))
+app.use(bodyparser.json())
+
+
+
+app.post('/',function(req, res){
+    
+    var test = "";
+    
+    //Case 1: When BOT is added to the ROOM
+    if(req.body.type == 'ADDED_TO_SPACE' && req.body.space.type == 'ROOM') {
+        text = `Thanks for adding me to ${req.body.space.type}`;

```

Comment:
The Python and Java implementations display the space name here, can you please do the same for consistency?

https://github.com/gsuitedevs/hangouts-chat-samples/blob/master/python/basic-bot/bot.py#L54